configs/JHEF435/config.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU        AT32F435G
+
+#define BOARD_NAME           JHEF435
+#define MANUFACTURER_ID      JHEF
+
+#define USE_ACC
+#define USE_GYRO
+
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO_SPI_ICM42688P
+
+#define USE_BARO
+#define USE_BARO_DPS310
+
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+
+#define USE_MAX7456
+
+#define BEEPER_PIN              PC15
+
+#define MOTOR1_PIN              PB6
+#define MOTOR2_PIN              PB7
+#define MOTOR3_PIN              PA3
+#define MOTOR4_PIN              PA2
+
+#define LED_STRIP_PIN           PB1
+
+#define UART1_TX_PIN            PA9
+#define UART2_TX_PIN            PA8
+#define UART3_TX_PIN            PB11
+#define UART5_TX_PIN            PB9    
+#define UART1_RX_PIN            PA10
+#define UART2_RX_PIN            PB0
+#define UART3_RX_PIN            PB10
+#define UART5_RX_PIN            PB8
+#define I2C2_SCL_PIN            PH2
+#define I2C2_SDA_PIN            PH3
+#define LED0_PIN                PC13
+#define SPI1_SCK_PIN            PA5
+#define SPI2_SCK_PIN            PB13
+#define SPI3_SCK_PIN            PB3
+#define SPI1_SDI_PIN            PA6
+#define SPI2_SDI_PIN            PB14
+#define SPI3_SDI_PIN            PB4
+#define SPI1_SDO_PIN            PA7
+#define SPI2_SDO_PIN            PB15
+#define SPI3_SDO_PIN            PB5
+#define ADC_VBAT_PIN            PA0
+#define ADC_CURR_PIN            PA1
+#define FLASH_CS_PIN            PA15
+#define MAX7456_SPI_CS_PIN      PB12
+#define GYRO_1_EXTI_PIN         PC14
+#define GYRO_1_CS_PIN           PA4
+
+#define TIMER_PIN_MAPPING       TIMER_PIN_MAP(0, PB1, 2, 3) \
+                                TIMER_PIN_MAP(1, PB6, 1, 0) \
+                                TIMER_PIN_MAP(2, PB7, 1, 1) \
+                                TIMER_PIN_MAP(3, PA3, 1, 2) \
+                                TIMER_PIN_MAP(4, PA2, 1, 4)
+
+#define ADC_INSTANCE            ADC1
+#define ADC1_DMA_OPT            11
+
+#define DEFAULT_BLACKBOX_DEVICE BLACKBOX_DEVICE_FLASH
+#define DEFAULT_DSHOT_BURST DSHOT_DMAR_AUTO
+#define DEFAULT_DSHOT_TELEMETRY DSHOT_TELEMETRY_ON
```

Comment:
Should use DShot bitbang instead of burst. DSHOT_TELEMETRY should not be enabled by default in the config. 
```suggestion
#define DEFAULT_DSHOT_BITBANG           DSHOT_BITBANG_ON
```

Reply:
I modify the file content here

Reply:
This was supposed to be default @sugaarK 

#define DEFAULT_DSHOT_TELEMETRY DSHOT_TELEMETRY_ON